lib/app/features/chat/presentation/pages/chat_people_card.dart

```diff
@@ -55,6 +74,60 @@ class ChatPeopleCard extends StatelessWidget {
       ),
     );
   }
+
+  Widget _buildBadgeWidget(List<UserDetailBadgeEntity> badges) {
+    if (badges.isEmpty) {
+      return const SizedBox.shrink();
+    }
+    var onlyInlineBadges = <UserDetailBadgeEntity>[];
+
+    for (final badge in badges) {
+      if (badge.style != 'inline-block') {
+        onlyInlineBadges.add(badge);
+      }
+    }
+
+    return Row(
+      children: onlyInlineBadges
+          .map((badge) => Padding(
+              padding: const EdgeInsets.only(left: 4.0),
+              child: UserBadgeWidget(
+                badgeDescription: badge.description,
+                badgeImageUrl: badge.imageUrl,
+                badgeName: badge.name,
+                badgePopUp: badge.popUp,
+                badgeShowDescription: badge.showDescription,
+              )))
+          .toList(),
+    );
+  }
+}
+
+Widget _buildCloseUser(List<UserDetailBadgeEntity> badges) {
+  if (badges.isEmpty) {
+    return const SizedBox.shrink();
+  }
+  var _emptyBadge = UserDetailBadgeEntity(
```

Comment:
```suggestion
  const  emptyBadge = UserDetailBadgeEntity(
```

1. Não temos o habito de utilizar '_' para variável dentro de uma função
2. Pelo o que entendi não será alterado, seria possível substituir o `var` por `const` ou `final`?